### ISSUE TITLE
(question) feature request: legacy office files

### ISSUE THREAD

**Issue by guillenotfound** [harshankur/officeParser#9]
Would it be possible to also support legacy Office files? If you know how to do it I can further implement it!

**Comment by harshankur** [comment]
Yeah it is in my TODO for this library but Microsoft has not released any official specification for the legacy office formats (doc, ppt, xls). There are a few specifications which are not 100% accurate. I will try to get my hands on them and give it a look. I will work on this too, I promise.

**Comment by guillenotfound** [comment]
Maybe this repo helps: https://github.com/nolze/msoffcrypto-tool
